### ISSUE TITLE
[Bug]: Embedding View Creation not Skipping Dropped Columns

### ISSUE THREAD

**Issue by jmartin127** [timescale/pgai#248]
### What happened?

When creating a new Vectorizer the function call fails, since it is trying to use a dropped column.

Column being pulled that shouldn't be is `pg.dropped.34`.  For example: `select attrelid, attname, attlen, attisdropped from pg_attribute where attname like '%dropped%'`.

Note: `attisdropped` can be used to identify (and exclude) dropped columns when generating the `create view` statement.

Screenshot 2024-11-22 at 1.46.51 PM

Error:

```
ERROR:  column s.........pg.dropped.34........ does not exist
LINE 41:     , s."........pg.dropped.34........"

    create view my_table_contents_embeddings as
    select
      t.embedding_uuid
    , t.chunk_seq
    , t.chunk
    , t.embedding
    , t.id
    , s.column1,
    , s.column2,
    , s."........pg.dropped.34........"
    from my_table_contents_embeddings_store t
    left outer join my_table s
    on (t.id = s.id)
```

### pgai extension affected

0.4.0

### pgai library affected

0.1.0

### PostgreSQL version used

16.4

### What operating system did you use?

Docker Container: postgres:16.4-alpine

### What installation method did you use?

Source

### What platform did you run on?

Google Cloud Platform (GCP)

### Relevant log output and stack trace

```bash
ERROR:  column s.........pg.dropped.34........ does not exist
LINE 41:     , s."........pg.dropped.34........"
```

### How can we reproduce the bug?

```bash
Steps to reproduce:


-- create a test table
CREATE TABLE testtab (
   id integer PRIMARY KEY,
   dropme text NOT NULL,
   val text NOT NULL
);

-- drop a column on the test table
ALTER TABLE testtab DROP dropme;

-- look over the attributes
SELECT attname, attnum, attisdropped
FROM pg_attribute
WHERE attrelid = 'testtab'::regclass
   AND attnum > 0
ORDER BY attnum;

-- try to create a vectorizer for the table
SELECT ai.create_vectorizer( 
   'testtab'::regclass,
    destination => 'testtab_contents_embeddings',
    embedding => ai.embedding_openai('text-embedding-3-small', 768),
    chunking => ai.chunking_recursive_character_text_splitter('val')
);

```
```


### Are you going to work on the bugfix?

None

**Comment by jmartin127** [comment]
Looks like the fix for this will be to update the `ai._vectorizer_create_view` function, by augmenting the `WHERE` clause to include `AND attisdropped = FALSE`

Here:

https://github.com/timescale/pgai/blob/1051dcd9484aacc60a1c1016b8de1c29b7645f27/projects/extension/sql/idempotent/012-vectorizer-int.sql#L229

**Comment by JamesGuthrie** [comment]
This was fixed in https://github.com/timescale/pgai/pull/214 and is released in pgai 0.4.1.
